examples/rest/stocks-mrk-aggregates_bars.py

```diff
@@ -0,0 +1,27 @@
+from polygon import RESTClient
```

Comment:
I see `-mrk-` in a lot of filenames, what does that mean?

Reply:
Yeah, sorry. That was my attempt to group these into "Market Data" aka mrk and "Reference Data" ref. Just so that when you're in the API docs you can see what belongs with what. Maybe I should change that to be a little more clear.

Reply:
Are you okay with the naming convention @mmoghaddam385 or you think I should change it?

Reply:
Ahh I see that makes sense, is `mrk` a common abbreviation for "market"/"market data"? "ref" is pretty straightforward so totally fine with that one. Maybe `stocks-pricing-<endpoint>`? Although that's no longer the same terminology that the API docs uses. TBH I don't really have a strong opinion either way, if you're looking at the API docs and see `mrk` and `ref` it'll be pretty clear what they mean so I'm fine with whatever you decide.

Reply:
Fixed. Just removed the `-mrk-` and `-ref-` from the filenames. Doesn't make sense to have them in there.